src/test/kotlin/no/nav/familie/ba/mottak/søknad/SøknadTreeWalkerTest.kt

```diff
@@ -27,11 +26,21 @@ class SøknadTreeWalkerTest {
 
     @Test
     fun `Test at verdier bevares og holdes sammen`() {
-        val navn = verdiliste.filter { it["label"] == "Barnets fulle navn" }.map { it["verdi"] }
-        val alder = verdiliste.filter { it["label"] == "alder" }.map { it["verdi"] }
-        assertTrue(
-                (navn[0] == "barn1" && alder[0] == "4 år")
-                        && (navn[1] == "barn2" && alder[1] == "1 år")
+        val barneliste = verdiliste[2]["verdiliste"] as List<Map<String, Any?>>
+        val barn1 = toMap(barneliste[0]["verdiliste"])
+        val barn2 = toMap(barneliste[1]["verdiliste"])
+        assertEquals(
+                Pair("barn1", "4 år"),
+                Pair(barn1["Barnets fulle navn"], barn1["alder"])
+        )
+        assertEquals(
+                Pair("barn2", "1 år"),
+                Pair(barn2["Barnets fulle navn"], barn2["alder"])
         )
     }
+
+    /** Gjør om liste av maps med nøkler: ("label", "verdi") til én map: ("label"-verdi -> "verdi"-verdi) */
+    fun toMap(list: Any?): Map<String, Any?> {
+        return (list as List<Map<String, Any?>>).map { Pair(it["label"] as String, it["verdi"]) }.toMap() // Må ha liste av par for å kjøpre toMap
```

Comment:
I stedet for å skrive `Pair(it["label"] as String, it["verdi"])`, så tror jeg dere kan skrive `it["label"] as String to it["verdi"]` 😄